dolphin/api/v1/storages.py

```diff
@@ -104,7 +105,7 @@ def create(self, req, body):
             raise exc.HTTPBadRequest(explanation=msg)
 
         try:
-            storage = self.driver_manager.register_storage(ctxt, access_info_dict)
+            storage = self.driver_api.register_storage(ctxt, access_info_dict)
```

Comment:
Hope you need to rebase tolatest master.. This change is merged.

Reply:
yes..this code is latest. i took after merge